lib/email_tracking_retriever.py

```diff
@@ -285,10 +287,20 @@ def clean_email_content(email_str) -> str:
   return email_str
 
 
-def log_in_if_necessary(driver: WebDriver):
+def log_in_if_necessary(driver: WebDriver, amazon_config):
   driver.get('https://www.amazon.com/gp/your-account/order-history/ref=ppx_yo_dt_b_orders')
   orders_containers = driver.find_elements_by_id('ordersContainer')
   if len(orders_containers) == 0:
     # likely a login screen
-    driver.find_element_by_css_selector('form[name="signIn"]')
-    input('Please log in to an Amazon account on the opened Chrome profile. Hit ENTER when done.')
+    if "email" in amazon_config and amazon_config["email"]:
```

Comment:
note my previous comment, this will break if the user doesn't have an 'amazon' section in their config. The code snippet I posted before should be the simplest type of logic to refer to the config, in that after checking that 'amazon' is in the config, we can just refer to config['amazon']['email'] (for instance)